snap/info.go

```diff
@@ -36,7 +36,10 @@ type PlaceInfo interface {
 	// Name returns the name of the snap.
 	Name() string
 
-	// MountDir returns the base directory of the snap.
+	// BaseDir returns the base directory for all versions of the snap.
+	BaseDir() string
```

Comment:
We had a base dir very recently, which got renamed because it was unclear. It doesn't feel much more clear now. Which base dir?